src/TLabel.cpp

```diff
@@ -67,9 +99,12 @@ void TLabel::setLeave(Host* pHost, const QString& func, const TEvent& args)
 
 void TLabel::mousePressEvent(QMouseEvent* event)
 {
-    if (event->button() == Qt::LeftButton) {
+    if (mMouseButtons.contains(event->button())) {
         if (mpHost) {
-            mpHost->getLuaInterpreter()->callEventHandler(mClick, mClickParams);
+            TEvent tmpClickParams = mClickParams;
+            tmpClickParams.mArgumentList.append(mMouseButtons[event->button()]);
```

Comment:
Tweak this to be `mMouseButtons.at(event->button())` - it's a better variant than `[]` because `[]` will create an element if it doesn't exist while all you were trying to do is read it. I know that's not possible given the `.contains()` you have above, it's just a good practice anyway.